src/test-helpers.ts

```diff
@@ -0,0 +1,30 @@
+import { act } from '@testing-library/react'
+
+/**
+ * Helper function to wait for React effects to complete in tests.
+ * This is commonly needed when testing React hooks that have async effects.
+ *
+ * @example
+ * ```typescript
+ * import { waitForEffects } from '../test-helpers'
+ *
+ * test('my test', async () => {
+ *   const { result } = renderHook(() => useMyHook())
+ *   await waitForEffects()
+ *   expect(result.current.isReady).toBe(true)
+ * })
+ * ```
+ */
+export const waitForEffects = () => act(() => setTimeout(() => {}, 0))
```

Comment:
```suggestion
import { act } from '@testing-library/react'
import { setTimeout } from 'timers/promises'

/**
 * Helper function to wait for React effects to complete in tests.
 * This is commonly needed when testing React hooks that have async effects.
 *
 * @example
 * ```typescript
 * import { waitForEffects } from '../test-helpers'
 *
 * test('my test', async () => {
 *   const { result } = renderHook(() => useMyHook())
 *   await waitForEffects()
 *   expect(result.current.isReady).toBe(true)
 * })
 * ```
 */
export const waitForEffects = () => act(() => setTimeout(0))
```